PCSX2_suite.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
+VisualStudioVersion = 15.0.27428.2005
```

Comment:
Why did you up the version number, VS 2015 uses the 14.0.25420.1 version number.

Reply:
I don't use VS 2015, but I had to edit the solution file to include the lua dependencies.  I'll manually edit the versions back